lib/pages/inventory.dart

```diff
@@ -1,8 +1,86 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:scibble/models/product.dart';
+import 'package:scibble/redux/inventory/actions.dart';
+import 'package:scibble/redux/store.dart';
+import 'package:scibble/theme/scibble_color.dart';
 
 class Inventory extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Text('Inventar');
+    return StoreConnector<AppState, _InventoryViewModel>(
+      converter: (store) => _InventoryViewModel(store.state.inventory,
+          getInventory: () => Future(store.dispatch(getInventory(store)))),
+      builder: (_, viewModel) => InventoryViewModel(
+        viewModel: viewModel,
+      ),
+      onInit: (store) => store.dispatch(getInventory(store)),
+    );
+  }
+}
+
+class InventoryViewModel extends StatelessWidget {
+  final _InventoryViewModel viewModel;
+
+  InventoryViewModel({Key key, this.viewModel}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return viewModel.inventory.length > 0
```

Comment:
When loading the inventory it will show the 'Refresh' button. This should be showed if it actually could not load anything.
I suggest changing the widget to be a statefulWidget, with a boolean value `loaded`. If false it should show a spinner. Much like it is done in `online_login.dart`.